### PR TITLE
[osci23] add scripts folder to lint-es  with rule 'no-var-requires' disabled

### PR DESCRIPTION
**Pull request by MadaniKK**
### Description
Added scripts folder to lint-es  with rule 'no-var-requires' disabled
Also uppressed ESLint errors on a per-line basis in some .jsfile
image

### Issues Resolved
resolved  issue #1096

### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x ] All tests pass
  - [x ] `yarn lint`
  - [x ] `yarn test-unit`
- [x ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by BSFishy — APPROVED** [review]
Looks great!

File: .eslintrc.js [hunk 1]
```diff
@@ -191,5 +191,11 @@ module.exports = {
         'react/prefer-stateless-function': 'off',
       },
     },
+    {
+      files: ['scripts/**/*.{ts,tsx,js}'], // Modify the pattern to match your specific directory
```
Comment (joshuarrrr) [inline .eslintrc.js:195]: Can we change this comment to be more specific about why we need this override? That will help future readers understand if it's still relevant or needed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -37,6 +37,8 @@
 - Bump TypeScript to v4.6.4 ([#879](https://github.com/opensearch-project/oui/pull/879))
 - Clean up `react-datepicker` package to remove unnecessary directories and files ([#1067](https://github.com/opensearch-project/oui/pull/1067))
 - Bump `@types/react` and `csstype` ([#1105](https://github.com/opensearch-project/oui/pull/1105))
+- Add `scripts` folder to lint-es script ([#1143](https://github.com/opensearch-project/oui/pull/1143))
+
```
Comment (joshuarrrr) [inline CHANGELOG.md:41]:
```suggestion
```

**Review by joshuarrrr — COMMENTED** [review]
Excellent work! Just a couple minor changes requested.

File: .eslintrc.js [hunk 3]
```diff
@@ -192,9 +192,10 @@ module.exports = {
       },
     },
     {
-      files: ['scripts/**/*.{ts,tsx,js}'], // Modify the pattern to match your specific directory
+      // Currently supressing the following rules for files in the scirpts directory
```
Comment (joshuarrrr) [inline .eslintrc.js:195]: spelling nits 😁 
```suggestion
      // Currently suppressing the following rules for files in the scripts directory
```

**Review by joshuarrrr — APPROVED** [review]

**Pull request merged**